docs/source/basics/working_with_files.md

```diff
@@ -49,6 +49,16 @@ def foo(fastq: LatchFile, output_dir: LatchDir) -> (LatchFile, LatchDir):
     # it will go back in your filesystem on the LatchBio console for now.
     return LatchFile("/root/foobar", "latch:///foobar.txt"), LatchDir(local_output_dir, output_dir.remote_path)
 ```
+_Writing to an existing remote LatchDir will only add or update files in it._ In the example below, it will leave the remote LatchDir untouched, rather than overwriting it and making it an empty directory.
```

Comment:
have a more specific example, e.g. add `test.txt` to a LatchDIr that exists and explain that it will add or overwrite `test.txt` only and not touch the other contents of the `LatchDir`